src/irmin-pack/IO.ml

```diff
@@ -44,6 +44,10 @@ module type S = sig
   val version : t -> string
 
   val sync : t -> unit
+
+  val close : t -> unit
+
+  val valid_fd : t -> bool
```

Comment:
maybe name this `is_valid`? we don't speak about `fd` in the rest of that signature